src/soundbase.cpp

```diff
@@ -184,14 +181,14 @@ QString CSoundBase::SetDev ( const QString strDevName )
             // to be able to access the ASIO driver setup for changing, e.g., the sample rate, we
             // offer the user under Windows that we open the driver setups of all registered
             // ASIO drivers
-            sErrorMessage = sErrorMessage + "<br/>" + tr ( "Do you want to open the ASIO driver setups?" );
+            sErrorMessage = sErrorMessage + "<br/>" + tr ( "Do you want to open the ASIO driver setup to change driver settings?" );
```

Comment:
Why is this confirmation here? Would there be some negative consequence of opening the setup?

Reply:
> Would there be some negative consequence of opening the setup

Probably not. However in some cases it won't help much since the device (not the driver) is somehow incompatible. The whole handling around this line is not perfect.

Reply:
The offer is only present for ASIO.

What's _far_ worse is line 191 (old and new) - they throw away this error, rather than letting CGenErr throw it, like for other drivers.

Reply:
See https://github.com/jamulussoftware/jamulus/issues/1812